test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,18 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
```

Comment:
Coding standard requires to include descriptive header comments for public methods